pages/index.js

```
@@ -40,6 +43,17 @@ export default function Home() {
 
   if (!data) return;
 
+  const filteredData = data.filter(
+    (item) =>
+      // fields to search by
+      (item.firstName &&
+        item.firstName.toLowerCase().includes(searchQuery.toLowerCase())) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)